src/app/_services/api/import-export.service.ts

```diff
@@ -55,61 +50,70 @@ export class ImportExportService {
     }
 
     uploadMCQuestion(question: Question): void {
-        const eventId = (typeof question.event === 'number') ? question.event : question.event?.id;
-        const input = {
-            title: question.title,
-            difficulty: question.difficulty,
-            course: question.course_name,
-            event: (String(eventId) === 'undefined') ? null : String(eventId),
-            text: question.text,
-            answer: question.answer,
-            category: String(this.categories.filter(x => x.pk === question.category)[0].pk),
-            variables: question.variables,
-            visible_distractor_count: question.visible_distractor_count,
-            choices: question.choices
-        };
-        this.questionService.postMultipleChoiceQuestion(input).subscribe((result) => {
-            if (result.success != false)
-                this.toastr.success('The Question has been added Successfully.');
+        this.categoryService.getCategory(question.category).subscribe(result => {
```

Comment:
I still don't understand why you are calling the category service. You are only using the id as far as I can see and you already have it.